lib/utils.ts

```diff
@@ -295,3 +295,8 @@ export const sortBytesArray = _array =>
 
 export const formatRound = (num, decimals = 2) =>
   isNaN(num) ? '----' : round(num, decimals).toLocaleString({}, { minimumFractionDigits: decimals });
+
+export const getVotingWeightCopy = (isVotingDelegate: boolean): string =>
```

Comment:
Instead of separating it into a function that is on a generic folder "utils" i would create a simple component under the relevant module "voting" or whatever.

Utils is a concept that we should discontinue, because leads to mix of responsabilities mixed

Reply:
Fair enough, moved the function to `lib/polling/`